src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx

```diff
@@ -0,0 +1,638 @@
+                
+                Subsequent uses: FutureStack
+            </td>
+            <td>Future Stack, Futurestack, Future stack
+            </td>
+        </tr>
+
+        <tr>
+            <td>NerdGraph*</td>
+            <td>our GraphQL API</td>
+            <td>First use: NerdGraph® (printed assets) or NerdGraph (digital assets)
+                
+                Subsequent uses: NerdGraph</td>
+            <td>Nerd Graph, Nerdgraph, nerdgraph, nerd graph
+            </td>
+        </tr>
+
+        <tr>
+            <td>Nerdlet*</td>
+            <td>component that make up New Relic One apps; a specific UI view represented by a React JavaScript package</td>
+            <td>First use: Nerdlet® (printed assets) or Nerdlet (digital assets)
+                
+                Subsequent uses: Nerdlet</td>
+            <td>nerdlet, NerdLet</td>
+        </tr>
+
+        <tr>
+            <td>Nerdpack*</td>
+            <td>component that make up New Relic One apps; the package containing all the files needed by that app</td>
+            <td>First use: Nerdpack® (printed assets) or Nerdpack (digital assets)
+                
+                Subsequent uses: Nerdpack
+            </td>
+            <td>nerdpack, NerdPack, Nerd Pack, nerd pack
+            </td>
+        </tr>
+
+        <tr>
+            <td>NerdStorage*</td>
+            <td>component that make up New Relic One apps; used to store and retrieve simple sets of data</td>
+            <td>First use: NerdStorage® (printed assets) or NerdStorage (digital assets)
+                
+                Subsequent uses: NerdStorage
+            </td>
+            <td>Nerdstorage, nerdstorage, Nerd Storage, Nerd storage, nerd storage
+            </td>
+        </tr>
+        
+        <tr>
+            <td>New Relic CodeStream</td>
+            <td>IDE extension that integrates with New Relic One</td>
+            <td>New Relic CodeStream (for the New Relic integration with CodeStream) or CodeStream (for just the CodeStream app)</td>
+            <td>New Relic CodeStream's, New Relic Code Stream, Code Stream
+            </td>
+        </tr>
+
+        <tr>
+            <td>New Relic Edge with Infinite Tracing</td>
+            <td>our fully-managed, tail-based, distributed tracing solution</td>
+            <td>First use: New Relic Edge with Infinite Tracing® (printed assets) or New Relic Edge with Infinite Tracing (digital assets)
+                
+                Subsequent uses: Infinite Tracing
+            </td>
+            <td>Infinite tracing, infinite tracing
+            </td>
+        </tr>
+        
+        <tr>
+            <td>New Relic Explorer</td>
+            <td>capability of New Relic One</td>
+            <td>First use: New Relic explorer
+                
+                Subsequent uses: New Relic Explorer
+                
+                Describing actions in the UI: Explorer</td>
+            <td>New Relic Explorer's
```

Comment:
Not sure if 'New Relic Explorer's Explorer' is a typo

Reply:
Oh I see. These are like different entries. this one was just lacking a comma like the others

Reply:
fixed.